scrapli/driver/base/base_driver.py

```diff
@@ -653,13 +661,15 @@ def _resolve_ssh_known_hosts(self, ssh_known_hosts: str) -> str:
         self.logger.debug("attempting to resolve 'ssh_known_hosts file'")
 
         resolved_ssh_known_hosts = ""
-
-        if Path(ssh_known_hosts).is_file():
-            resolved_ssh_known_hosts = str(Path(ssh_known_hosts))
-        elif Path("~/.ssh/known_hosts").expanduser().is_file():
-            resolved_ssh_known_hosts = str(Path("~/.ssh/known_hosts").expanduser())
-        elif Path("/etc/ssh/ssh_known_hosts").is_file():
-            resolved_ssh_known_hosts = str(Path("/etc/ssh/ssh_known_hosts"))
+        if ssh_known_hosts == "" and transport == "system":
+            resolved_ssh_known_hosts = BaseTransport.SSH_SYSTEM_CONFIG_MAGIC_STRING
+        else:
```

Comment:
same "flatten-y" comment here (if possible/easy, if not we can ignore -- and sorry just looking quickly so maybe this is a bad comment!)

Reply:
Changed it the same way as above :-)